app/src/main/java/com/guruthedev/instagram/ui/fragments/ProfileFragment.kt

```diff
@@ -27,7 +27,9 @@ class ProfileFragment : Fragment() {
         binding.fullNameProfileFrag.text = fullName.toString()
         binding.profileFragmentUsername.text = username.toString()
         binding.bioProfileFrag.text = bio.toString()
+        binding.postImages.setOnClickListener {
 
+        }
         return binding.root
     }
 }
```

Comment:
I see "No new line at end of file" warning with many files. Please add new line.